packages/safe-core-sdk-types/src/types.ts

```diff
@@ -59,6 +58,26 @@ export interface SafeTransaction {
   encodedSignatures(): string
 }
 
+export interface Transaction {
+  to: string
+  value: string
+  data: string
+  from?: string
+  gas?: number | string
+  gasLimit?: number | string
+  gasPrice?: number | string
+  maxFeePerGas?: number | string
+  maxPriorityFeePerGas?: number | string
+  nonce?: number
```

Comment:
This type can have different parameters or types depending on the provider library used (ethers/web3).

At least it should reuse the `TransactionOptions` at line 81 and we should review if there isn't another type covering this.

Reply:
Done!

I am reusing the `TransactionOptions` interface to define the `Transaction` interface

```typescript
export type Transaction = TransactionBase & TransactionOptions

interface TransactionBase {
  to: string
  value: string
  data: string
}

export interface TransactionOptions {
  from?: string
  gas?: number | string
  gasLimit?: number | string
  gasPrice?: number | string
  maxFeePerGas?: number | string
  maxPriorityFeePerGas?: number | string
  nonce?: number
}
```